addons/point_of_sale/static/tests/tours/PosComboTour.js

```diff
@@ -82,3 +82,46 @@ registry.category("web_tour.tours").add("PosComboPriceTaxIncludedTour", {
         // the split screen is tested in `pos_restaurant`
     ],
 });
+
+registry.category("web_tour.tours").add("PosComboChangeFP", {
+    test: true,
+    url: "/pos/ui",
```

Comment:
This is not required actually. The full url is already specified when starting the tour from the python test